platformio.ini

```diff
@@ -61,7 +60,8 @@ build_src_filter =
 	${env.build_src_filter}
 	+<../_Boards/Atmel>
 lib_deps = 
-	${env.lib_deps}	
+	${env.lib_deps}
+	arduino-libraries/Servo @ 1.1.8
```

Comment:
Instead of putting the arduino library in each arduino env, how about creating a new `lib_amtel_deps` section then including that as `${env.lib_amtel_deps}`?

Reply:
Ah, very good idea. I will change this.

Reply:
Maybe pair it with a `lib_rp2040_deps` too, even though there's just one rp2040 platform?

Reply:
Hmhm, there are no additional libs for the Pico.